x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -1355,9 +1363,11 @@ metricbeat.modules:
   # Password to use when connecting to PostgreSQL. Empty by default.
   #password: pass
 
-#----------------------- Prometheus Typed Metrics Module -----------------------
+#------------------------------ Prometheus Module ------------------------------
+# Metrics collected from a Prometheus endpoint
 - module: prometheus
   period: 10s
+  metricsets: ["collector"]
```

Comment:
Does this need to change as part of this PR?

Reply:
No, in that it isn't part of this change.  But Yes in that `mage config` generates it.  I don't know why https://github.com/elastic/beats/pull/40411 didn't already pull this change in.